sbnobj/SBND/Blip/BlipDataTypes.h

```diff
@@ -0,0 +1,174 @@
+    int       TPC           = -9;     // TPC ID
+    float     Time          = -9;     // time of particle interaction
+    int       TimeTick      = -9;     // time tick
+    float     DriftTime     = -9;     // drift time [us]
+    float     Energy        = 0;      // energy dep [MeV]
+    int       DepElectrons  = 0;      // deposited electrons
+    int       NumElectrons  = 0;      // electrons reaching wires
+    int       LeadG4ID      = -9;     // lead G4 track ID
+    int       LeadG4Index   = -9;     // lead G4 track index
+    int       LeadG4PDG     = -9;     // lead G4 PDG
+    float     LeadCharge    = -9;     // lead G4 charge dep
+    TVector3  Position;               // XYZ position
+    mif_t     G4ChargeMap;          
+    mif_t     G4PDGMap;
+  };
+
+  struct HitInfo {
+    int   hitid         = -9;
+    int   cryo          = -9;
+    int   tpc           = -9;
+    int   plane         = -9;
+    int   wire          = -9;
+    int   chan          = -9;
+    float amp           = -9;
+    float rms           = -9;
+    int   trkid         = -9;
+    int   shwrid        = -9;
+    int   clustid       = -9;
+    int   blipid        = -9;
+    bool  ismatch       = false;
+    float integralADC    = -999;     // [ADCs] from integral
+    float sigmaintegral = -999;
+    float sumADC        = -999;     // [ADCs] from sum 
+    float charge        = -999;     // [e-]
+    float peakTime      = -999999;
+    float driftTime     = -999999;  // [tick]
+    float gof           = -9;
+    int   g4trkid       = -9;
+    int   g4pdg         = -999;
+    int   g4charge      = -999;     // [e-]
+    float g4frac        = -99;      
+    float g4energy      = -999;     // [MeV]
+  };
+  
+  struct HitClust {
+    int     ID              = -9;
+    bool    isValid         = false;
+    int     CenterChan      = -999;
+    int     CenterWire      = -999;
+    bool    isTruthMatched  = false;
+    bool    isMerged        = false;
+    bool    isMatched       = false;
+    int     DeadWireSep     = 99;
+    int     Cryostat        = -9;
+    int     TPC             = -9;
+    int     Plane           = -9;
+    int     NHits           = -9;
+    int     NWires          = -9;
+    float   ADCs            = -999;
+    float   Amplitude       = -999;
+    float   Charge          = -999;
+    float   SigmaCharge     = -999;
+    float   TimeTick        = -999;
+    float   Time            = -999;
+    float   StartHitTime    = -999;
+    float   EndHitTime      = -999;
+    float   StartTime       = -999;
+    float   EndTime         = -999;
+    float   Timespan        = -999;
+    float   RMS             = -999;
+    int     StartWire       = -999;
+    int     EndWire         = -999;
+    int     NPulseTrainHits = -9;
+    float   GoodnessOfFit   = -999;
+    int     BlipID          = -9;
+    int     EdepID          = -9;
+    si_t    HitIDs;
+    si_t    Wires;
+    si_t    Chans;
+    si_t    G4IDs;
```

Comment:
Also here, document what is each of these. Are they just lists with no particular order? or are any of these paired so that the same index describes the same entity?